packages/react/src/patterns/sub-patterns/CardLink/CardLink.js

```diff
@@ -58,7 +61,11 @@ const CardLink = ({
  */
 function optionalContent(content) {
   return !content ? null : (
-    <p className={`${prefix}--card-link__content`}>{content}</p>
+    <div
+      className={`${prefix}--card-link__content`}
+      dangerouslySetInnerHTML={{
+        __html: markdownToHtml(content, { bold: false, allowHtml: true }),
```

Comment:
What does the `allowHtml` do?

Reply:
From the specs:
```Determines if HTML markup is permitted. If disabled, HTML tags will be stripped out.```
By default, this is false. I think we should keep it off, honestly.

Reply:
oh ok I thought we wanted it though,

`Italic markdown compiles to HTML` <- acceptance criteria from the ticket

Reply:
there is another story to update the markdown utility to handle paragraphs, but otherwise we are only converting italics and nothing else: https://github.com/carbon-design-system/ibm-dotcom-library/issues/1256

Reply:
gotcha ok, removed `allowHtml`